cmd/cluster-node-tuning-operator/main.go

```diff
@@ -72,7 +76,10 @@ func init() {
 
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(tunedv1.AddToScheme(scheme))
-	utilruntime.Must(mcov1.AddToScheme(scheme))
+	if !config.InHyperShift() {
+		klog.Infof("Running in NOT hypershift")
```

Comment:
nit: I'd avoid negatives phrased like this. What about `running on regular non-hypershift cluster`?

Reply:
In addition, what's the drawback of always register the mcov1 APIs?

Reply:
In hypershift we are using also [this other mcAPI](https://github.com/jlojosnegros/cluster-node-tuning-operator/blob/98c7a75a15c31956544323702569b6aec329201e/pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go#L31) and I there is a conflict when we register them both